roles/katello_devel/meta/main.yml

```diff
@@ -13,5 +14,8 @@ dependencies:
       - foreman-installer-katello
     foreman_installer_options_internal_use_only:
       - "--disable-system-checks"
+      - "--katello-devel-use-rvm=false"
+      - "--katello-devel-use-scl-ruby=true"
```

Comment:
Do we need these options? Aren't they the defaults?

Reply:
I thought about this as well, but I assumed we stored answers for the devel scenario. Now it looks like forklift always overwrites answers so that's pointless. We might as well add the option `dont_save_answers: true` to remove any illusions they persist.